ckanext/dalrrd_emc_dcpr/assets/css/dalrrd-emc-dcpr.css

```diff
@@ -1071,8 +1071,11 @@ footer {
 }
 
 .metadata-record-thumbnail img{
-  width: 100%;
+  width: 150px;
+  height: 150px;
   vertical-align: middle;
+  object-fit: cover !important;
```

Comment:
@voogt , remember to use !important only as a last resort. It's not good practice, and it could become difficult to maintain in the future.

Sorry, just passing by...